sdks/python/container/boot.go

```diff
@@ -380,26 +380,26 @@ func installSetupPackages(files []string, workDir string, requirementsFiles []st
 	pkgName := "apache-beam"
 	isSdkInstalled := isPackageInstalled(pkgName)
 	if !isSdkInstalled {
-		return fmt.Errorf("Apache Beam is not installed in the runtime environment. If you use a custom container image, you must install apache-beam package in the custom image using same version of Beam as in the pipeline submission environment. For more information, see: the https://beam.apache.org/documentation/runtime/environments/.")
+		return fmt.Errorf("apache-beam is not installed in the runtime environment. If you use a custom container image, you must install apache-beam package in the custom image using same version of Beam as in the pipeline submission environment. For more information, see: the https://beam.apache.org/documentation/runtime/environments/")
 	}
 	// Install the Dataflow Python SDK and worker packages.
 	// We install the extra requirements in case of using the beam sdk. These are ignored by pip
 	// if the user is using an SDK that does not provide these.
 	if err := installSdk(files, workDir, sdkSrcFile, acceptableWhlSpecs, false); err != nil {
-		return fmt.Errorf("failed to install SDK: %v", err)
+		return fmt.Errorf("failed to install SDK (this may be unrecoverable): %v", err)
```

Comment:
Could we add more context here? Either a reason of why each of these errors are unrecoverable, or even better a link to a doc explaining it?

As a user, it wouldn't really be obvious to me what to do with this (many errors in a pipeline "may be unrecoverable" if you don't have more context about them)

Reply:
This will also be paired with logged output from PIP at error saying that a dependency failed to be installed once the buffered logger is implemented so that context will be there. A doc explaining what we mean by "unrecoverable" would be good though, something like a "Common Unrecoverable Errors" FAQ

Reply:
Yeah, something like https://cloud.google.com/dataflow/docs/guides/common-errors (but probably a Beam page)

Reply:
It can be super brief